src/map/skill.cpp

```diff
@@ -10605,8 +10652,14 @@ int skill_castend_nodamage_id (struct block_list *src, struct block_list *bl, ui
 		if (src != bl && rnd()%100 < 20 * skill_lv) {
 			int x = src->x, y = src->y;
 
-			if (hd)
-				skill_blockhomun_start(hd,skill_id,skill_get_time2(skill_id,skill_lv));
+#ifdef RENEWAL
+			if (hd != nullptr)
+			{
+				skill_blockhomun_start(hd, skill_id, skill_get_time2(skill_id, skill_lv));
+				if (hd->master != nullptr)
+					skill_blockpc_start(hd->master, skill_id, skill_get_time2(skill_id, skill_lv));
+			}
+#endif
```

Comment:
```suggestion
			if (hd != nullptr)
			{
				skill_blockhomun_start(hd, skill_id, skill_lv);
			}
```
Here you reference `skill_get_time2()`, was this a mistake? The skill_db still uses `Duration2`.